### DISCUSSION TITLE
v0.11.9

### DISCUSSION THREAD

**Discussion by fcakyon** [obss/sahi#801] [Announcements]
## What's Changed
* update yolov5 version in ci by @fcakyon in https://github.com/obss/sahi/pull/797
* fix torch dependency in HuggingfaceDetectionModel by @fcakyon in https://github.com/obss/sahi/pull/798
* update installation, support latest transformers version by @fcakyon in https://github.com/obss/sahi/pull/799

**Full Changelog**: https://github.com/obss/sahi/compare/0.11.8...0.11.9

<hr /><em>This discussion was created from the release <a href='https://github.com/obss/sahi/releases/tag/0.11.9'>v0.11.9</a>.</em>

**Comment by wickeat** [comment]
Hi @fcakyon, I'm wondering if the `yolov8onnx.py` model can also have its [torch dependency](https://github.com/obss/sahi/blob/7e36595bfa4b2ebf59117866d89496db6c002bad/sahi/models/yolov8onnx.py#L9) moved from the module level to the function level where it's actually used? This has already been done in your fix in https://github.com/obss/sahi/pull/798.
